src/util/well_known_sids.c

```diff
@@ -34,6 +34,14 @@
 #define BUILTIN_SID_PREFIX_LEN (sizeof(BUILTIN_SID_PREFIX) - 1)
 #define BUILTIN_DOM_NAME "BUILTIN"
 
+#define AUTH_SID_PREFIX "S-1-5-64-"
+#define AUTH_SID_PREFIX_LEN (sizeof(AUTH_SID_PREFIX) - 1)
+#define AUTH_DOM_NAME "AUTH"
+
+#define OTHER_SID_PREFIX "S-1-5-65-"
+#define OTHER_SID_PREFIX_LEN (sizeof(OTHER_SID_PREFIX) - 1)
+#define OTHER_DOM_NAME "OTHER"
```

Comment:
Hi,

I think `OTHER` is wrong here, something like `THIS ORG` might be better. However, I agree with your concern that putting S-1-5-64-* and S-1-5-65-* into `NT AUTHORITY` would be better since Windows is doing the same, e.g. with Powershell you can see:

```
PS C:\Users\Administrator> $objSID = New-Object System.Security.Principal.SecurityIdentifier('S-1-5-64-10')
PS C:\Users\Administrator> $objSID.Translate([System.Security.Principal.NTAccount])

Value
-----
NT AUTHORITY\NTLM Authentication
```

or 

```
PS C:\Users\Administrator> $objSID = New-Object System.Security.Principal.SecurityIdentifier('S-1-5-65-1')
PS C:\Users\Administrator> $objSID.Translate([System.Security.Principal.NTAccount])

Value
-----
NT AUTHORITY\This Organization Certificate
```

S-1-5-32-* is handle special on Windows as well:

```
PS C:\Users\Administrator> $objSID = New-Object System.Security.Principal.SecurityIdentifier('S-1-5-32-546')
PS C:\Users\Administrator> $objSID.Translate([System.Security.Principal.NTAccount])                                         
Value
-------                                                                                           
BUILTIN\Guests
```

HTH

bye,
Sumit

Reply:
Thanks for all this information @sumit-bose .  I think it is better to align to what Windows does.